src/components/Todo.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+const styles = {
+    textDecoration: 'italic'
+};
+const Todo = props => {
+    return (
+        <div style={props.todo.completed ? styles : null}
```

Comment:
The prop 'todo.completed' does not exist, nor is it passed in upon creation of each todo inside the TodoList class. If you want to have access to this function, you need to pass it when you are maping over ```this.state.todos``` inside the TodoList, I suggest console.logging the props of TodoList so you know how to access the function passed down from App.